### PR TITLE
Setting Page- Improvement needed

### PR DESCRIPTION
**Pull request by snuff4**
Worked on Issue #6,

1. The UI needs a lot of improvement, I wanted it like the Spotify Dark theme, but failed :/

2. The functions are working, except the switch which clicks all of them.

3. Tried finding the conflicting file.

simulator screen shot - iphone x - 2018-02-26 at 03 22 24_iphonexspacegrey_portrait

simulator screen shot - iphone x - 2018-02-26 at 03 22 29_iphonexspacegrey_portrait

simulator screen shot - iphone x - 2018-02-26 at 03 27 50_iphonexspacegrey_portrait

simulator screen shot - iphone x - 2018-02-26 at 03 22 34_iphonexspacegrey_portrait

simulator screen shot - iphone x - 2018-02-26 at 03 22 40_iphonexspacegrey_portrait

simulator screen shot - iphone x - 2018-02-26 at 03 21 21_iphonexspacegrey_portrait

### CODE REVIEW

**Comment by snuff4** [comment]
The conflict was due to the reason that I moved the Main storyboard into a New Folder "View", for all the future storyboards.

File: AmahiAnywhere/AmahiAnywhere/Presentation/DashboardViewController.swift [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 import UIKit
 
 class DashboardViewController: BaseUIViewController {
     
-  
     override func viewDidLoad() {
         super.viewDidLoad()
         // Check access token in the debug area
-       print("ACCESS TOKEN: \(LocalStorage.shared.getAccessToken()!)")
+        print("ACCESS TOKEN: \(LocalStorage.shared.getAccessToken()!)")
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere/Presentation/DashboardViewController.swift]: And again with print()

**Comment by snuff4** [comment]
I made few changes, with all the buttons working and made a preview [video](https://github.com/snuff4/ios/blob/Setting-Improvement/WhatsApp%20Video%202018-03-02%20at%2015.32.52.mp4). Please review it and request changes @cpg @codedentwickler

**Comment by cpg** [comment]
Well, if we're not going to answer questions raised in reviews repeatedly, and the code continues to have conflicts, we cannot continue, sorry. Thanks for your interest.

**Comment by snuff4** [comment]
@cpg Okay, it's my fault that I replied on the IRC instead regarding the print() review.
And there was Only one conflict arising due to NSURL changed to URL, because we don't use NSURL anymore.

Thankyou! I learned a lot in these days.

**Pull request closed without merging**